gcp/config.ts

```diff
@@ -0,0 +1,18 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as utils from "./utils";
+
+let config = new pulumi.Config();
+
+export const GcpCloud = "gcp";
+export const LightbendNamespace = "lightbend";
+
+export const cloud = config.get<string>("cloud") || GcpCloud;
+export const zone = config.get<string>("gcp:zone") || "europe-west1-b";
```

Comment:
Don't think that there should be a default?

Reply:
+1 .. we can remove `region` but `zone` has to be set either at the provider level (using `gcloud`) or at the config level using `pulumi config set`. Otherwise we get the following error:

```
Error: invocation of gcp:container/getEngineVersions:getEngineVersions returned an error: invoking gcp:container/getEngineVersions:getEngineVersions: 1 error occurred:
    	* Cannot determine zone: set in this resource, or set provider-level zone.
```

Should we go for a default value of zone or fail fast and ask the user to set one ? @RayRoestenburg

Reply:
fail fast SGTM